src/test/java/lesson6Task2/FileUtilsReadFromFileTest.java

```diff
@@ -0,0 +1,15 @@
+package lesson6Task2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FileUtilsReadFromFileTest {
+
+    @Test
+    void readFromFile() {
+        String filePath = "src\\main\\resources\\DataReader.txt";
+        String content = FileUtilsReadFromFile.readFromFile(filePath);
+        System.out.println("Content of a file: \n" + content);
```

Comment:
We always have to add some assertion inside test, like: Assertions.assertEquals("Expected text from file", textFromFile)